src/SigasiProjectCreator/convertDotFtoCsv.py

```diff
@@ -0,0 +1,60 @@
+from SigasiProjectCreator.DotF.parseFile import parse_dotf
+import sys
+import os
+import csv
+
+def rebase_file(filename, dotfdir):
+    hdlpath = os.path.expandvars(filename)
+    if not os.path.isabs(hdlpath):
+        hdlpath = os.path.join(dotfdir, hdlpath)
+    hdlpath = os.path.normpath(hdlpath)
+    return hdlpath
+
+
+def convertDotFtoCsv(filename):
+    if not os.path.isfile(filename):
+        print("*ERROR* File " + filename + " does not exist")
+        sys.exit(1)
```

Comment:
I always find it useful to use distinct exit codes for different exit conditions and to document which error results in which code

Reply:
Exit codes are now unique and documented at the start of the file in which they are used.